src/TimeSeries/widget/TimeSeries.ts

```diff
@@ -0,0 +1,191 @@
+import * as dojoDeclare from "dojo/_base/declare";
+import * as mxLang from "mendix/lang";
+import * as WidgetBase from "mxui/widget/_WidgetBase";
+
+import { createElement } from "react";
+import { render, unmountComponentAtNode } from "react-dom";
+
+import { HeightUnit, SeriesConfig, WidthUnit } from "../TimeSeries.d";
+import { DataPoint, TimeSeries, WidgetProps } from "./components/TimeSeries";
+
+export class TimeSeriesWrapper extends WidgetBase {
+    // Parameters configured in the Modeler  
+    private xAxisLabel: string;
+    private xAxisFormat: string;
+    private yAxisLabel: string;
+    private yAxisFormat: string;
+    private seriesConfig: SeriesConfig[];
+    private width: number;
+    private height: number;
+    private widthUnit: WidthUnit;
+    private heightUnit: HeightUnit;
+
+    // Internal variables. Non-primitives created in the prototype are shared between all widget instances.
+    private contextObject: mendix.lib.MxObject;
+    private dataLoaded: boolean;
+    private dataStore: any;
+    private handle: number;
+
+    private createProps(): WidgetProps {
+        return {
+            dataLoaded: this.dataLoaded,
```

Comment:
This attribute is removed, since it's obsolete.
It was originally used for controlling re-rendering once all data has been fetched.